udata/api/__init__.py

```diff
@@ -227,6 +227,8 @@ def extract_name_from_path(path):
     """
     base_path, query_string = path.split('?')
     infos = base_path.strip('/').split('/')[2:]  # Removes api/version.
+    if base_path == '/api/1/':  # The API root endpoint redirects to swagger doc.
+        return safe_unicode('Api_doc')
```

Comment:
Can we change this strange half-snake :snake: half-upside-down-camel :camel: case? Or could we just use `apidoc` ? :p